### PR TITLE
Do, channel update

### PR DESCRIPTION
**Pull request by Pecaquito**
Main Dominican Channel update

streams/do.m3u

### CODE REVIEW

**Review by QHegazy — COMMENTED** [review]
### Non-Working Links
1. **Antena 7**  
   **Link:** [https://alba-do-antena7-antena7.stream.mediatiquestream.com/index.m3u8](https://alba-do-antena7-antena7.stream.mediatiquestream.com/index.m3u8)  
   **Issue:** This link does not work at all. It seems to be dead or unreachable.

2. **TVS Live**  
   **Link:** [https://imagenuniversaltv.net:3741/live/tvslive.m3u8](https://imagenuniversaltv.net:3741/live/tvslive.m3u8)  
   **Issue:** Similar to the previous link, this one is also not functional. It does not load or play any content.

### Partially Working Links
1. **Telecentro**  
   **Link:** [http://51.81.93.29:8084/hls/telecentro.m3u8](http://51.81.93.29:8084/hls/telecentro.m3u8)  
   **Issue:** This link works intermittently. The stream freezes frequently and then resumes playing, which disrupts the viewing experience. While it's better than the non-working links, it still needs improvement.

**Comment by BellezaEmporium** [comment]
> ### Non-Working Links
> 
>     1. **Antena 7**
>        **Link:** https://alba-do-antena7-antena7.stream.mediatiquestream.com/index.m3u8
>        **Issue:** This link does not work at all. It seems to be dead or unreachable.
> 
>     2. **TVS Live**
>        **Link:** https://imagenuniversaltv.net:3741/live/tvslive.m3u8
>        **Issue:** Similar to the previous link, this one is also not functional. It does not load or play any content.
> 
> 
> ### Partially Working Links
> 
>     1. **Telecentro**
>        **Link:** http://51.81.93.29:8084/hls/telecentro.m3u8
>        **Issue:** This link works intermittently. The stream freezes frequently and then resumes playing, which disrupts the viewing experience. While it's better than the non-working links, it still needs improvement.

1) VPN to DO
2) Not 24/7.
3) No issues on my side.

File: streams/do.m3u [hunk 1]
```diff
@@ -14,9 +14,13 @@ https://ss9.domint.net:3040/ame_str/amecanal47/playlist.m3u8
 #EXTINF:-1 tvg-id="AnimeZoneTV.do",Anime Zone TV (480p)
 http://animezonetv.net/hls/stream.m3u8
 #EXTINF:-1 tvg-id="Antena7.do",Antena 7 (480p) [Not 24/7]
-https://d1841z6ti5nz2v.cloudfront.net/Antena7DO/34cdaa501a0217ea217d8f4f083c1728.sdp/playlist.m3u8
+https://alba-do-antena7-antena7.stream.mediatiquestream.com/index.m3u8
+#EXTINF:-1 tvg-id="Antena7.do",Antena 7 
+https://ssh101.bozztv.com/ssh101/rctvhd21/chunks.m3u8
 #EXTINF:-1 tvg-id="Antena21.do",Antena 21 (480p) [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/do.m3u:20]:
```suggestion
#EXTINF:-1 tvg-id="Antena21.do",Antena 21 (480p) [Not 24/7] [Geo-blocked]
```

File: streams/do.m3u [hunk 2]
```diff
@@ -14,9 +14,13 @@ https://ss9.domint.net:3040/ame_str/amecanal47/playlist.m3u8
 #EXTINF:-1 tvg-id="AnimeZoneTV.do",Anime Zone TV (480p)
 http://animezonetv.net/hls/stream.m3u8
 #EXTINF:-1 tvg-id="Antena7.do",Antena 7 (480p) [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/do.m3u:16]:
```suggestion
#EXTINF:-1 tvg-id="Antena7.do",Antena 7 (480p) [Not 24/7] [Geo-blocked]
```

File: streams/do.m3u [hunk 3]
```diff
@@ -244,19 +248,19 @@ http://51.81.93.239:8084/hls/tmicro.m3u8
 #EXTINF:-1 tvg-id="Telemilenio.do",Telemilenio (720p) [Not 24/7]
 http://cm.hostlagarto.com:8081/Telemilenio/Telemilenio.myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="Telenord8.do",Telenord 8 (1080p) [Not 24/7]
-http://newyorkstream.ddns.net:1935/telenord8/telenord8/playlist.m3u8
+https://rdn.essastream.com:3170/live/telenord8live.m3u8
 #EXTINF:-1 tvg-id="Telenord12.do",Telenord 12 (720p) [Not 24/7]
-http://newyorkstream.ddns.net:1935/telenord12/telenord12/playlist.m3u8
+https://vdo1.streamgato.us:3345/live/telenord12live.m3u8
 #EXTINF:-1 tvg-id="Telenord10.do",Telenord Canal 10 (720p) [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/do.m3u:254]:
```suggestion
#EXTINF:-1 tvg-id="Telenord10.do",Telenord 10 (720p) [Not 24/7]
```

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**